run/src/main/kotlin/com/cosmotech/run/service/RunServiceImpl.kt

```diff
@@ -296,13 +297,14 @@ class RunServiceImpl(
       runId: String,
       runDataQuery: RunDataQuery
   ): QueryResult {
-
     checkInternalResultDataServiceConfiguration()
-    getRun(organizationId, workspaceId, runnerId, runId)
+    // getRun(organizationId, workspaceId, runnerId, runId)
 
     val runtimeDS =
         DriverManagerDataSource(
-            "jdbc:postgresql://$host:$port/$runId", readerStorageUsername, readerStoragePassword)
```

Comment:
Do we go back to hardcode `localhost`, I don't think this is was we want (wrong merge conflict resolve on your part?)